egs/mgb5/s5/local/nnet3/run_tdnn.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+# cat exp/nnet3/tdnn_1a/decode_dev/scoring_kaldi/best_wer
```

Comment:
update the comments (WER and info)
add a comment from where the system was copied from

Reply:
done in tuning/run_tdnn_1a.sh

Reply:
+ I think this should be a link to tuning/run_tdnn_1a.sh
did you make a copy by accident?

Reply:
removed file and now symbolic link